web/src/components/dashboard/SidebarCardGeneric.tsx

```diff
@@ -1,106 +1,98 @@
 import { Content } from "@/components/Content";
-import { SecondaryButton } from "@/components/njwds-extended/SecondaryButton";
+import { PrimaryButton } from "@/components/njwds-extended/PrimaryButton";
 import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
 import { Icon } from "@/components/njwds/Icon";
 import { useSidebarCards } from "@/lib/data-hooks/useSidebarCards";
-import { MediaQueries } from "@/lib/PageSizes";
 import { SidebarCardContent } from "@/lib/types/types";
-import useMediaQuery from "@mui/material/useMediaQuery";
-import { ReactElement, ReactNode } from "react";
+import { ReactElement } from "react";
 import { ModifiedContent } from "../ModifiedContent";
 
 type Props = {
   card: SidebarCardContent;
   bodyText: string;
   headerText: string | undefined;
-  preBodyContent?: ReactNode;
+  preBodySpanButtonText: string | undefined;
+  preBodyButtonOnClick?: () => void;
   ctaOnClick?: () => void;
-  layout?: "row" | "column";
 };
 
 export const SidebarCardGeneric = (props: Props): ReactElement => {
   const { hideCard } = useSidebarCards();
-  const isMobile = useMediaQuery(MediaQueries.isMobile);
-
-  const layout = props.layout ?? "column";
 
   const closeSelf = async (): Promise<void> => {
     await hideCard(props.card.id);
   };
 
+  const sideCardClassDictionary: { [key: string]: string } = {
+    "formation-nudge": "primary-gradient",
+    "funding-nudge": "primary-gradient-reverse",
+    "registered-for-taxes-nudge": "secondary-gradient",
+    "go-to-profile": "secondary-gradient-reverse",
+    "not-registered-existing-account": "tertiary-gradient",
+    "not-registered": "tertiary-gradient-reverse",
+  };
+
   const ctaButton = (
-    <SecondaryButton
-      isColor="primary"
+    <PrimaryButton
+      isColor="white"
       onClick={props.ctaOnClick}
       isRightMarginRemoved={true}
       dataTestId={`cta-${props.card.id}`}
       isFullWidthOnDesktop={true}
     >
       {props.card.ctaText}
-    </SecondaryButton>
-  );
-
-  const stackedBodyContent = (
-    <>
-      <Content>{props.bodyText}</Content>
-      {props.ctaOnClick && props.card.ctaText && (
-        <div className="margin-top-205 flex flex-justify-center desktop:flex-justify-end">{ctaButton}</div>
-      )}
-    </>
-  );
-
-  const rowBodyContent = (
-    <div className="flex fdr fac">
-      <Content className="margin-right-1">{props.bodyText}</Content>
-      {props.ctaOnClick && props.card.ctaText && <div className="width-100">{ctaButton}</div>}
-    </div>
+    </PrimaryButton>
   );
 
   return (
     <>
       <div
-        className={`border radius-md border-${props.card.borderColor} margin-bottom-3`}
+        className={`border radius-md sideBarCardBorder margin-bottom-3 ${
```

Comment:
recommend using a color specific class name

Reply:
your adding a border then removing it ? `border radius-md sideBarCardBorder`

Reply:
done